src/handlers.js

```diff
@@ -65,6 +65,27 @@ async function singlePackageListing(req, res, timecop) {
       og_image_height: 600,
     }});
   } catch(err) {
+    let status_to_display = false; // Since the status is ignored if not a number,
+    // we initialize as boolean to no-op in the case we don't find a proper status
+
+    const validStatus = (val, key) => {
+      if (val?.response?.[key] && typeof val.response[key] === "boolean" && val.response[key]) {
```

Comment:
I would suggest streamlining this a tad.

A very shortened version of what I believe is an equivalent check:

```suggestion
      if (val?.response?.[key] === true) {
```

Or maybe this:

```suggestion
      if (typeof val?.response?.[key] === "boolean" && val.response[key]) {
```

^ if the convention of explicitly checking `typeof` is important, to signal to the reader "this is doing type checking," and to be less mistake prone with `true` (boolean) vs `"true"` (string) (kind of like "TypeScript but with less tooling," heh).

Third suggestion, why not:

```suggestion
      if (val?.response?.[key] && typeof val.response[key] === "boolean") {
```

Reply:
Honestly, none of my suggestions here are more reasable while still explicitly checking `typeof ... === "boolean"`, so I think it's basically more readable as-is. I'll mark this as resolved.